src/resources/font.js

```diff
@@ -3,9 +3,9 @@ var WebFont = require('webfontloader');
 const fonts = () => {
   WebFont.load({
     google: {
-      families: [ 'Open Sans', 'Poppins', 'Lora' ]
+      families: [ 'Open Sans', 'Poppins: 100,300,500,600,900', 'Lora: 300,600,900' ]
```

Comment:
This is way too many fonts to load each time you add a number thats another font that will need to be loaded